### DISCUSSION TITLE
UI in chrome is not friendly

### DISCUSSION THREAD

**Discussion by dayuanyuan1989** [dani-garcia/vaultwarden#3104] [General]
![image](https://user-images.githubusercontent.com/8370129/210736740-4c44cd2a-0674-4285-8539-bc3d11ec9f5e.png)

The content  filling with the background may be friendly.

**Comment by BlackDex** [comment]
These kind of questions are better suited for the Bitwarden clients repo or Bitwarden community forum.

Vaultwarden does not maintain the clients.

**Reply by dayuanyuan1989** [reply]
I accessed vaultwarden server to get the UI in browser(chrome) , I think that it can fixed in vaultwarden project.

**Reply by dayuanyuan1989** [reply]
![image](https://user-images.githubusercontent.com/8370129/210921596-8b68d123-bbc8-4006-8e48-047dfc869231.png)

**Reply by BlackDex** [reply]
I'm afraid not. We only make some changes to let the web-vault work nicely with the Vaultwarden backend. All other GUI items should be reported to Bitwarden. Also that it will benefit all users of Bitwarden, not only Vaultwarden.

**Reply by dayuanyuan1989** [reply]
OK. thx.
